### ISSUE TITLE
Not all problems are reported

### ISSUE THREAD

**Issue by beeblebrox3** [silverbulleters/sonarqube-inject-vsc#16]
I have a file with two problems. On vscode with the extension installed, only one is reported. When I use the CLI (downloaded by the extension) I see the two problems.

On PHPStorm I see the two problems too.
![sonar](https://cloud.githubusercontent.com/assets/767624/25433474/d7d89fe8-2a5e-11e7-9e98-6dcc45505eaa.png)

**Comment by nixel2007** [comment]
Could you provide console output from sonarlint-cli with additional `--reportType console" params?

**Comment by beeblebrox3** [comment]
```
$ /home/luis/.vscode/extensions/silverbulleters.sonarqube-inject-1.1.5/tools/sonarlint-cli/bin/sonarlint --src "app/module/expedicao/service/business/ExpedirCargaBO.php" --reportType console
INFO: Java 1.8.0_121 Oracle Corporation (64-bit)
INFO: Linux 4.4.0-21-generic amd64
INFO: Connected mode (wms-mv.com)
INFO: Using storage for server 'sbx' (last update 26/04/17 08:44)
INFO: Index files
INFO: 1 files indexed
INFO: 1 source files to be analyzed
INFO: 1/1 source files have been analyzed
INFO: No PHPUnit test report provided (see 'sonar.php.tests.reportPath' property)
INFO: No PHPUnit unit test coverage report provided (see 'sonar.php.coverage.reportPath' property)
INFO: No PHPUnit integration test coverage report provided (see 'sonar.php.coverage.itReportPath' property)
INFO: No PHPUnit overall coverage report provided (see 'sonar.php.coverage.overallReportPath' property)
INFO: {/home/luis/projects/wms/app/module/expedicao/service/business/ExpedirCargaBO.php} {MINOR} {null:null - null:null} {php:S113} {Add a new line at the end of this file.}
{/home/luis/projects/wms/app/module/expedicao/service/business/ExpedirCargaBO.php} {MAJOR} {104:64 - 104:71} {php:S881} {Extract this increment or decrement operator into a dedicated statement.}


-------------  SonarLint Report  -------------

          2 issues (1 file analyzed)

          1 major
          1 minor

-------------------------------------------


INFO: ------------------------------------------------------------------------
INFO: EXECUTION SUCCESS
INFO: ------------------------------------------------------------------------
INFO: Total time: 2.825s
INFO: Final Memory: 7M/173M
INFO: ------------------------------------------------------------------------

```

**Comment by nixel2007** [comment]
Looks like [this regexp](https://github.com/silverbulleters/sonarqube-inject-vsc/blob/master/src/features/lintProvider.ts#L74) can't handle null in string line and coloumn. Thanks for bug report, i'll try to fix this soon!

**Comment by nixel2007** [comment]
Check the 1.2.0 release, please!

**Comment by beeblebrox3** [comment]
Is working well now. Thanks :)
